etna/ensembles/direct_ensemble.py

```diff
@@ -25,6 +25,13 @@ class DirectEnsemble(EnsembleMixin, SaveEnsembleMixin, BasePipeline):
     For each point in the future, forecast of the ensemble is forecast of base pipeline with the shortest horizon,
     which covers this point.
 
+    See Also
+    --------
+    Pipeline -> etna.pipeline.Pipeline:
```

Comment:
There was a misunderstanding, there should be:
```
etna.pipeline.Pipeline:
    Pipeline that forecasts values in one iteration using a model.
etna.pipeline.AutoRegressivePipeline:
    Pipeline that forecasts values in several iterations using a model.
```

Reply:
Oh, I should have been more attentive)